ui.pyw

```diff
@@ -115,7 +115,12 @@ class FoodFinderUi(Frame, object): #TODO Rename
         user = self.box_value.get().strip()
         if not user:
             return
-        self.food_finder.add_user(user)
+
+        try:
+            self.food_finder.add_user(user)
+        except Exception: #TODO catch specific exceptions and handle appropriately
```

Comment:
This is fine, just a general FYI if you don't care about the type you are catching and you are doing nothing with the caught instance then you don't have to specify the type.
Can be just "except:"
This also has an interesting sideeffect that you are not affecting the stacktrace if you reraise